Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/Components/KeyStage4PerformanceTables/default.cshtml

```diff
@@ -292,23 +301,24 @@
     </tr>
     </thead>
     <tbody class="govuk-table__body">
+        @Html.Raw(@Html.KeyStageDataRow())
```

Comment:
Then your usage of them can be like this - https://github.com/DFE-Digital/prepare-academy-conversions/blob/67f040ea9d83dea7766153917b6ced09ce99adfa/Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/GeneralInformation/ViabilityIssues.cshtml#L14